cohttp-async/bin/cohttp_server_async.ml

```diff
@@ -50,7 +50,7 @@ let serve ~info ~docroot ~index uri path =
         serve_file ~docroot ~uri
       | `No | `Unknown -> (* Do a directory listing *)
         Sys.ls_dir file_name
-        >>= Deferred.List.map ~f:(fun f ->
+        >>= Deferred.List.map ~how:`Sequential ~f:(fun f ->
```

Comment:
Why's this Sequential? It looks like it could as easily be concurrent, given the function is just going over a set of independent files.

Reply:
Apologies for the delay, fixed.